src/Domain/File.php

```diff
@@ -112,7 +112,7 @@ public function process(): void
                 $this->reportActiveSniffClass($sniff);
 
                 try {
-                    $sniff->process($this, $stackPtr);
+                    @$sniff->process($this, $stackPtr);
```

Comment:
This also ignore `Throwable` exceptions I think, can you double check? Thanks buddy!

Reply:
The way the `@` works is that it basically changes the error reporting to 0.

I just tested with these two examples
```
<?=(preg_match(PRIVILEGES['something'],$user['privileges'])?'<a href="..">aa</a>':'')?>
```
and
```
<?php

/*
```

We get the two sniffs saying unparsable code, but we have no warnings thrown 😄 

![image](https://user-images.githubusercontent.com/5870441/63494683-82468200-c4be-11e9-9ea0-78e33da832ec.png)
![image](https://user-images.githubusercontent.com/5870441/63494708-912d3480-c4be-11e9-916b-4d92a4e583e6.png)